### ISSUE TITLE
sphereface

### ISSUE THREAD

**Issue by jianbaba** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#1]
您好，请问：如果use arccosine instead of the double angle formula，这样复现的结果是否与sphereface论文中的结果一致？
